src/main/resources/application.yml

```
@@ -0,0 +1,18 @@
+        password: ${DB_PASSWORD:123456}
+    main:
+        allow-bean-definition-overriding: true
+    jpa:
+        database-platform: org.hibernate.dialect.PostgreSQLDialect
+        hibernate:
+            ddl-auto: none
+    liquibase:
+        change-log: ${CHANGE_LOG:classpath:db/changelogs/master-changelog.xml}
+        enabled: true
+server:
+    port: ${PORT:8386}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)